src/test/packages/34-manifest-with-symlink/manifests/resources/img

```diff
@@ -0,0 +1 @@
+../img
```

Comment:
I don't see a file called `img` in the parent directory though in git (which would be `manifests`)

Reply:
I guess because there was no file in it.  I'm pushing up a text file, which is probably better to make sure that recursing works.